frontend/src/lib/WebsocketConnection.tsx

```diff
@@ -342,14 +349,14 @@ export class WebsocketConnection {
       PING_MINIMUM_RETRY_PERIOD_MS,
       PING_MAXIMUM_RETRY_PERIOD_MS,
       this.args.onRetry,
-      this.args.setHostAllowedOrigins,
+      this.args.setAllowedOriginsResp,
       userCommandLine
     )
 
     this.stepFsm("SERVER_PING_SUCCEEDED")
   }
 
-  private connectToWebSocket(): void {
+  private connectToWebSocket(): Promise<void> {
```

Comment:
Not a strong objection, but: why isn't this an `async` function?

Reply:
No reason in particular -- I probably just somewhat absentmindedly wrote this code to return the promise directly rather than equivalently making this an async function

It looks like writing functions as `async` is more common throughout the codebase, so will change to doing that